docs/collections/dynamo/create-dynamo-table.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Create a Dynamo Table
+sidebar_position: 10
+---
+
+Dynamo Table collection is available for AWS Dynamo Database users. Macrometa’s platform can be used as a data store to acquire Global data distribution within milliseconds. To switch to GDN, the user need to change the connection URL, AccessKey, and SecretKey. AWS Dynamo DB SDK and CLI are compatible. Geo Distribution is set to Global by default.
+
+## Create a Dynamo Table Collection  with the Console
+
+Follow these instructions to create a new collection using the GDN console web UI.
+
+1. Log in to your Macrometa account.
```

Comment:
```suggestion
1. Log into your Macrometa account.
```

Reply:
The original is actually correct. Weird, I know.

Reply:
> The original is actually correct. Weird, I know.

I always thought it was more of a preference thing, but either way works for me.  Will remember this!